internal/controllers/machine/machine_controller.go

```diff
@@ -343,13 +345,45 @@ type scope struct {
 	// Machine. It is set after reconcileInfrastructure is called.
 	infraMachine *unstructured.Unstructured
 
+	// infraMachineNotFound is true if getting the infra machine object failed with an IsNotFound err
+	infraMachineIsNotFound bool
+
 	// bootstrapConfig is the BootstrapConfig object that is referenced by the
 	// Machine. It is set after reconcileBootstrap is called.
 	bootstrapConfig *unstructured.Unstructured
+
+	// bootstrapConfigNotFound is true if getting the BootstrapConfig object failed with an IsNotFound err
+	bootstrapConfigIsNotFound bool
+
+	// node is the Kubernetes node hosted on the machine.
+	node *corev1.Node
+}
+
+func (r *Reconciler) reconcileMachineOwnerAndLabels(_ context.Context, s *scope) (ctrl.Result, error) {
+	// If the machine is a stand-alone one, meaning not originated from a MachineDeployment, then set it as directly
+	// owned by the Cluster (if not already present).
```

Comment:
```suggestion
	// If the machine is a stand-alone Machine, then set it as directly
	// owned by the Cluster (if not already present).
```
nit: current version is incomplete